content/06.discussion.md

```diff
@@ -106,18 +104,22 @@ As a result, several opportunities for innovation arise: understanding the cause
 Unfortunately, uncertainty quantification techniques are underutilized in the computational biology communities and largely ignored in the current deep learning for biomedicine literature.
 Thus, the practical value of uncertainty quantification in biomedical domains is yet to be appreciated.
 
-### Interpretation
+### Interpretability
 
-As deep learning models achieve state-of-the-art performance in a variety of domains, there is a growing need to make the models more interpretable.
-Interpretability matters for two main reasons.
-First, a model that achieves breakthrough performance may have identified patterns in the data that practitioners in the field would like to understand.
-However, this would not be possible if the model is a black box.
-Second, interpretability is important for trust.
-If a model is making medical diagnoses, it is important to ensure the model is making decisions for reliable reasons and is not focusing on an artifact of the data.
-A motivating example of this can be found in Caruana et al. [@tag:Caruana2015_intelligible], where a model trained to predict the likelihood of death from pneumonia assigned lower risk to patients with asthma, but only because such patients were treated as higher priority by the hospital.
-In the context of deep learning, understanding the basis of a model's output is particularly important as deep learning models are unusually susceptible to adversarial examples [@tag:Nguyen2014_adversarial] and can output confidence scores over 99.99% for samples that resemble pure noise.
+As deep learning models achieve state-of-the-art performance in a variety of domains, there is a growing need to make the models more interpretable. There are several important reasons to care about interpretability.
 
-As the concept of interpretability is quite broad, many methods described as improving the interpretability of deep learning models take disparate and often complementary approaches.
+Firstly, a model that achieves breakthrough performance may have identified patterns in the data that practitioners in the field would like to understand.
```

Comment:
One sentence per line here, but I also might note that one of the major hazards of noninterpretable models with high predictive accuracy would be learning patterns in the data that capture the data selection process as opposed to the underlying data generating process (i.e., ascertainment biases, etc).